pontifical-university-lateranense-ius.csl

```diff
@@ -0,0 +1,577 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" default-locale="it-IT">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Pontifical University Lateranense IUS (Italian)</title>
```

Comment:
Since you've fixed this style to Italian, it might be better name this style in Italian as well ("Pontificia Università Lateranense" instead of "Pontifical University Lateranense").